### DISCUSSION TITLE
Return Oriented Programming 解题讨论

### DISCUSSION THREAD

**Discussion by wumingzhilian** [HUSTSeclab/software-security-dojo#26] [General]
我们在这里讨论 Return Oriented Programming 模块中遇到的问题

**Comment by RutingZhang0429** [comment]
1.1这一题是要构造cat /flag吗，感觉构造过程特别复杂

**Reply by wumingzhilian** [reply]
orw 或者 execve都可以

**Reply by RutingZhang0429** [reply]
<img width="302" alt="image" src="https://github.com/HUSTSeclab/software-security-dojo/assets/129151845/40216073-19b7-4f81-9bb7-5bbfdd1354ca">
我是这样写的，但是没有结果

**Reply by wumingzhilian** [reply]
![image](https://github.com/HUSTSeclab/software-security-dojo/assets/69114599/7d9ccc99-c14f-4c6e-8d8f-9a67c27b0379)
0xb 是哪个系统调用号呢？

**Reply by wumingzhilian** [reply]
这个地方有问题
